### ISSUE TITLE
Update to latest apispec

### ISSUE THREAD

**Issue by bugeats** [mathewmarcus/apispec-chalice#1]
Several versions behind now, and it's causing conflicts.

**Comment by izieg-lo3** [comment]
I have found `apispec-chalice` to be very useful, but the deeper I get into it I'm finding it difficult to use because it's hard to find the right documentation for older versions of `apispec`.

**Comment by mathewmarcus** [comment]
@bugeats @izieg-lo3 you are absolutely right, `apispec-chalice` does need to be updated to be compatible with the newer versions of `apispec`. Unfortunately, I do not have the capacity to do this at the moment. Would either of you be interested in submitting a PR? If not, I'll get to it as soon as I can.

**Comment by jordan-dimov** [comment]
Any update on this?
